deps.edn

```diff
@@ -2,7 +2,8 @@
  {"opensaml" {:url "https://build.shibboleth.net/nexus/content/repositories/releases/"}}
 
  :deps
- {org.clojure/spec.alpha              {:mvn/version "0.3.218"}
+ {org.clojure/clojure                 {:mvn/version "1.11.1"}
```

Comment:
Since this is a library we don't want to specify the Clojure version here since that will potentially cause conflicts with the downstream projects using this. If you want to specify it for hacking on it locally put it in a `:dev` profile or something like that